package.json

```diff
@@ -84,16 +84,17 @@
     "screwdriver-artifact-bookend": "^1.0.1",
     "screwdriver-build-bookend": "^2.0.1",
     "screwdriver-config-parser": "^3.10.0",
-    "screwdriver-data-schema": "^16.14.7",
+    "screwdriver-data-schema": "^17.0.0",
     "screwdriver-datastore-sequelize": "^2.0.0",
```

Comment:
Need to bump to `3.0.0` because of this: https://github.com/screwdriver-cd/datastore-sequelize/pull/20